README.md

```diff
@@ -153,6 +155,18 @@ COMMAND SUPPLIED
     git-xargs --version
     ```
 
+### Installion option 2: Run go get 
+
+1. **Run go get to install the latest release of git-xargs**
```

Comment:
This should have as a requisite that you have Go coding env up and running.

Reply:
Added in b983994